pkg/fanal/analyzer/analyzer.go

```diff
@@ -74,13 +76,19 @@ type analyzer interface {
 	Version() int
 	Analyze(ctx context.Context, input AnalysisInput) (*AnalysisResult, error)
 	Required(filePath string, info os.FileInfo) bool
+	Description() string
```

Comment:
This method is not used anywhere. The analyser type can be obtained using the Type method.

Reply:
Yes, but we can do it in another PR.